docker-compose.yml

```
@@ -0,0 +1,11 @@
+services:
+  app:
+    build:
+      dockerfile_inline: |
+        FROM ruby:2.6-bullseye
+        COPY Gemfile Gemfile.lock act_as_api_client.gemspec .
+        RUN bundle install
+    working_dir: /app
+    volumes:
+      - ./:/app
+    command: /bin/sh -c "while sleep 1000; do :; done"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)